components/automate-ui/src/app/modules/infra-proxy/create-org-modal/create-org-modal.component.html

```diff
@@ -0,0 +1,44 @@
+<chef-modal [visible]="visible" (closeModal)="closeEvent()">
+  <h2 slot="title">Create New Org</h2>
+  <div class="flex-container">
+    <form [formGroup]="createForm">
+      <div class="name-margin"> 
+        <chef-form-field>
+          <label>
+            <span class="label">Org Name <span aria-hidden="true">*</span></span>
```

Comment:
Make the same change I saw you did earlier (here and others in this file):
```suggestion
            Org Name <span aria-hidden="true">*</span>
```